exchange/binance.go

```
@@ -109,10 +109,11 @@ func (self *Binance) UpdatePrecisionLimit(pair common.TokenPair, symbols []Binan
 	}
 }
 
+//UpdatePairsPrecision update precision for trading for pairs
 func (self *Binance) UpdatePairsPrecision() {
 	exchangeInfo, err := self.interf.GetExchangeInfo()
 	if err != nil {
-		log.Printf("Get exchange info failed: %s\n", err)
+		log.Panicf("Get exchange info from binance failed: %s", err.Error())
```

Maybe this function should return an error instead?

---

This function can return error. However, this function runs when core started and it will stop the core if cannot get info for its supported tokens from exchanges.

---

Great, please change it to return an error, the caller will decide if it want to panic or not.